.github/workflows/windows_nightlies.yml

```diff
@@ -69,4 +69,16 @@ jobs:
         run: |
           . $env:USERPROFILE\Miniconda3\shell\condabin\conda-hook.ps1
           conda activate curr_py
-          make ${{matrix.command}}
+          make ${{matrix.command}}]
```

Comment:
What's up with the right square bracket?

Reply:
Ah that's an accident. Good catch!